conda_libmamba_solver/solver.py

```diff
@@ -821,7 +789,7 @@ def _export_solved_records(
                     # ^ Do not try to unlink virtual pkgs, virtual eggs, etc
                     continue
                 if record.fn == filename:  # match!
-                    out_state.records.pop(name, None, reason="Unlinked by solver")
```

Comment:
That seems to be the case elsewhere in the patch, will we lose details for debugging?

Reply:
These were "reasons" attached to the state models. I never used them for any debugging and always focused on the tasks sent to the solver. In this particular case we could add an extra `log.debug()` call if you feel this is important. However, since there's a warning when we don't find a record, I assumed it would only add noise to the already verbose logs.

Reply:
Sounds good.